### ISSUE TITLE
Evaluate automated accessibility tooling

### ISSUE THREAD

**Issue by mekline** [lookit/lookit-api#1021]
# TL;DR

We have had (and can repeat) a one-off accessibility evaluation from MIT's office, but don't currently have any practices or tools that support ongoing maintenance of accessible html and related issues.  There exist libraries and CI/CD tools that could help with this, and we should pick something to implement that makes sense for our scope.

# Narrative

As a user, I want an accessible experience of the Lookit website!

As a developer, I don't want to have to manually scan for problem areas where automated tools could do the same work. 

# Acceptance Criteria
A precise description of how/when this issue will be considered closed, e.g.:

- [ ] Some potential libraries, tools, or procedures have been explored
- [ ] Select which ones we'll implement and create issues to do so

# Implementation Notes

Blog post on accessibility linters etc. : https://blog.testproject.io/2022/02/07/accessibility-testing-best-practices-in-ci-cd/
MIT's digital accessibility page: https://studentlife.mit.edu/das/accessibility/digital-accessibility
